submit.py

```diff
@@ -357,6 +359,8 @@ def write_general_header(self, f, mem=3000, walltime_hours=14, disk=1,
             self.write_option(f, "--partition=%s" % self.config['Cluster']["partition"])
         self.write_option(f, "--time=%d:00:00" % walltime_hours)
         if self.config["Mode"]["debug"]:
+            if not os.path.isdir("%s/out/"%os.getcwd()):
+                os.mkdir("%s/out/"%os.getcwd())
```

Comment:
and while one is changing % for os.path.join, what about changing % by format? :-)